src/components/DpCard/DpCard.vue

```diff
@@ -8,6 +8,7 @@
       </span>
       <dp-contextual-help
         v-if="headingTooltip !== ''"
+        class="inline-block"
```

Comment:
I'd suggest to apply this change directly in [DpContextualHelp](https://github.com/demos-europe/demosplan-ui/blob/79705eaee1c4dfbbcf86b672d0bee95954c0b15d/src/components/DpContextualHelp/DpContextualHelp.vue#L4). In almost all cases when DpContextualHelp is used, we do not want the tooltip icon to be rendered on a new line, but be positioned next to the element it is explaining.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/829/commits/c813ef8db669786a38fdfe10f51f825fc24e35f7